pyads/__init__.py

```diff
@@ -32,6 +32,7 @@
     PLCTYPE_DT,
     PLCTYPE_DWORD,
     PLCTYPE_INT,
+    PLCTYPE_LINT,
```

Comment:
I noticed that this was missing from the imports. Not strictly needed for this PR, so I can remove it from here if you'd prefer it in a separate PR.

Reply:
Never mind. This is just a small import. No need to make a separate PR for it.